src/code.cloudfoundry.org/policy-server/store/migrations/migrator.go

```diff
@@ -33,7 +36,21 @@ type Migrator struct {
 }
 
 func (m *Migrator) PerformMigrations(driverName string, migrationDb MigrationDb, maxNumMigrations int) (int, error) {
-	migrationsToPerform, err := m.MigrationsProvider.MigrationsToPerform()
+	var mysql57 bool
+	if driverName == "mysql" {
+		row := migrationDb.QueryRow("SELECT VERSION()")
+		// if no rows are returned, we're definitely not mysql 5.7, so short circuit out
+		if row != nil {
+			var version string
+			err := row.Scan(&version)
+			if err != nil {
+				return 0, fmt.Errorf("Unable to detect MySQL Version: %s", err)
+			}
+			mysql57 = strings.HasPrefix(version, "5.7.")
```

Comment:
Do you know what this returns for all versions? 

Do it ever return just "5.7"?

Reply:
Can you add a comment with an example of what the version looks like?

Reply:
Comment added. Returns 5.7.x or 8.x.x for the versions i've tested. I couldn't find a docker image of 5.7.0, so it's possible that returned just 5.7, but that seems unlikely (and also why is anyone still running 5.7.0 at this point).